swagger/transforms.go

```diff
@@ -61,6 +66,10 @@ func ValidateErrors(s spec.Swagger) error {
 				}
 			}
 
+			if successResponseCount > 1 {
+				return fmt.Errorf("can only define one success type (statusCode < 400) for %s", op.ID)
+			}
+
```

Comment:
Want to just add validation that we don't have any 3XX responses? We can always remove later if we see a logical reason to have one.